src/Google/Ads/GoogleAds/Util/FieldMasks.php

```diff
@@ -163,6 +163,7 @@ private static function buildPaths(
                     // beginning.
                     (!is_null($original) && method_exists($original, $hasser)
                         && $original->$hasser() != $modified->$hasser())
+                    || (is_null($original) && !$modified)
```

Comment:
Why do we check for `$modified` here?
Why not `$modifiedValue`?

I ask because above, we also obtained the `$modifiedValue`, which is the result of `$modified->$getter()`, so probably we don't expect this `$modified` to be a primitive value.

Reply:
You are right, I changed.